docs/libertyRun.md

```diff
@@ -0,0 +1,43 @@
+## libertyRun task  
+Start a Liberty server in the foreground. The server instance will be automatically created if it does not exist.  
+**NOTE!** For proper server shutdown and shutdown console messages when terminating `libertyRun` with a Ctrl-C, you **must** run `libertyRun` like this:  
```

Comment:
**NOTE:** For proper server shutdown and to view shutdown console messages when terminating `libertyRun` with a Ctrl-C, use the `--no-daemon` option on the `libertyRun` task: